spec/views/idv/shared/_document_capture.html.erb_spec.rb

```diff
@@ -92,4 +94,12 @@
       end
     end
   end
+  describe 'view variables sent correctly' do
+    it 'sends doc_auth_selfie_capture to the FE' do
+      render_partial
+      expect(rendered).to have_css(
+        "#document-capture-form[data-doc-auth-selfie-capture='#{doc_auth_selfie_capture}']",
```

Comment:
There's something wrong here, but I haven't been able to track down a fix yet.

Reply:
My guess is that it's something to do with the way the hash is being rendered in the view html. If I put a `binding.pry` in and look at `rendered`, I see:

`data-doc-auth-selfie-capture=\"{&quot;enabled&quot;:false}\"`

There probably is a way to get the hash to display better. Another options is we change the way it renders to:

`"#document-capture-form[data-doc-auth-selfie-capture='#{doc_auth_selfie_capture[:enabled]}']"`

Reply:
Thanks! Yeah, it was an issue with escaping, fixed in this [commit](https://github.com/18F/identity-idp/pull/9536/commits/b87e5bcecf279bc740ba69ef12e418e0300def6f)!